subsys/logging/log_output_syst.c

```diff
@@ -672,17 +675,84 @@ void log_output_string_syst_process(const struct log_output *log_output,
 
 	MIPI_SYST_PRINTF(&log_syst_handle, severity, str);
 }
+#else
+static void hexdump2_print(struct log_msg2 *msg,
+			  const struct log_output *output)
+{
+	char *buf;
+	size_t length;
+	uint32_t severity = level_to_syst_severity(log_msg2_get_level(msg));
+
+	buf = log_msg2_get_data(msg, &length);
+	MIPI_SYST_WRITE(&log_syst_handle, severity, DATA_ID, buf, length);
+}
 
-void log_output_hexdump_syst_process(const struct log_output *log_output,
```

Comment:
You can't remove it. It is used by LOG V1 syst implementation. `log_output_hexdump` calls it.

Reply:
It was by accident I think. Have put this back.